packages/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java

```diff
@@ -151,7 +151,7 @@ public void onCatalystInstanceDestroy() {
     mMultiFactorSessions.clear();
   }
 
-  @ReactMethod
+    @ReactMethod
```

Comment:
formatting changes shouldn't be mixed in with functional changes, and formatting should be controlled by google-java-format and our `yarn lint:android` run script should run it

```suggestion
  @ReactMethod
```